gcloud/storage/api.py

```diff
@@ -143,6 +135,9 @@ def create_bucket(bucket_name, project=None, connection=None):
 
     This implements "storage.buckets.insert".
 
+    If the bucket already exists, will raise
+    :class:`gcloud.exceptions.Conflict`.
+
```

Comment:
It doesn't get raised by the code here, but rather by the methods called.

This would be like listing `ValueError` every time `json.loads` is called (or even more exotic possible errors).

If there is no `raise` statement, then the `:raises:` statement doesn't fit.

----

It's not a big deal either way. I'm happy to change it if you have strong feelings. The above was my logic.